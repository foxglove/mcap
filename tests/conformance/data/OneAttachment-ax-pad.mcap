version https://git-lfs.github.com/spec/v1
oid sha256:ae8a2bdbf3a58cc554e315ecce52ce5bb817d17bba30e3eb2b17eff0ccc2f935
size 244
