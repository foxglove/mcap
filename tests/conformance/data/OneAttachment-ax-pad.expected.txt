Header profile="" library=""
Attachment name=myfile recordTime=2 contentType=application/octet-stream data=<010203>
AttachmentIndex recordTime=2 attachmentSize=3 name=myfile contentType=application/octet-stream offset=28 attachmentRecordLength=81
Footer summaryStart=125 summaryOffsetStart=0
